libhl-crypto/amcl/src/aes.rs

```diff
@@ -0,0 +1,628 @@
+/*
```

Comment:
@mikelodder7 Is it the final the decision to copy-paste origin AMCL to HL/URSA? May be we should use https://github.com/milagro-crypto/amcl/tree/master/version3/rust ?

Reply:
No, it will be an iterative process. The next PR I make will fix this as z-mix also has amcl, so we need to dedup it. I wouldn't block this PR for this reason. I'll let you know about it when I raise the next PR.

Reply:
See my branch [here](https://github.com/mikelodder7/ursa/tree/z-mix)